src/dialogs/more-info/components/siren/ha-more-info-siren-advanced-controls.ts

```diff
@@ -0,0 +1,225 @@
+import "../../../../components/ha-textfield";
+import "../../../../components/ha-control-button";
+import "../../../../components/ha-select";
+import "../../../../components/ha-list-item";
+import type { HaMdDialog } from "../../../../components/ha-md-dialog";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { supportsFeature } from "../../../../common/entity/supports-feature";
+import { SirenEntityFeature } from "../../../../data/siren";
+import { haStyle } from "../../../../resources/styles";
+
+@customElement("ha-more-info-siren-advanced-controls")
+class MoreInfoSirenAdvancedControls extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() _stateObj?: HassEntity;
+
+  @state() _tone?: string;
+
+  @state() _volume?: number;
+
+  @state() _duration?: number;
+
+  @query("ha-md-dialog") private _dialog?: HaMdDialog;
+
+  public showDialog({ stateObj }: { stateObj: HassEntity }) {
+    this._stateObj = stateObj;
+  }
+
+  public closeDialog(): void {
+    this._dialog?.close();
+  }
+
+  private _dialogClosed(): void {
+    this._stateObj = undefined;
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  render() {
+    if (!this._stateObj) {
+      return nothing;
+    }
+    const supportsTones =
+      supportsFeature(this._stateObj, SirenEntityFeature.TONES) &&
+      this._stateObj.attributes.available_tones;
+    const supportsVolume = supportsFeature(
+      this._stateObj,
+      SirenEntityFeature.VOLUME_SET
+    );
+    const supportsDuration = supportsFeature(
+      this._stateObj,
+      SirenEntityFeature.DURATION
+    );
+    return html`
+      <ha-md-dialog
+        open
+        @closed=${this._dialogClosed}
+        aria-labelledby="dialog-light-color-favorite-title"
+        .getOpenAnimation=${getMobileOpenFromBottomAnimation}
+        .getCloseAnimation=${getMobileCloseToBottomAnimation}
+      >
+        <ha-dialog-header slot="headline">
+          <ha-icon-button
+            slot="navigationIcon"
+            @click=${this.closeDialog}
+            .label=${this.hass.localize("ui.common.close")}
+            .path=${mdiClose}
+          ></ha-icon-button>
+          <span slot="title" id="dialog-light-color-favorite-title"
+            >${this.hass.localize(
+              "ui.components.siren.advanced_controls"
+            )}</span
+          >
+        </ha-dialog-header>
+        <div slot="content">
+          <div class="options">
+            ${supportsTones
+              ? html`
+                  <ha-select
+                    .label=${this.hass.localize("ui.components.siren.tone")}
+                    @closed=${stopPropagation}
+                    @change=${this._handleToneChange}
+                    .value=${this._tone}
+                  >
+                    <ha-list-item value="">
+                      ${this.hass.localize("ui.common.default")}
+                    </ha-list-item>
```

Comment:
`ha-select` shows the placeholder on top of the value if the value is empty:
![image](https://github.com/user-attachments/assets/a0e07d32-5800-437c-ae14-1af8262f0d8e)

you would need to add some kind of default value